Modules/CTP/include/CTP/RawDataQcTask.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   RawDataQcTask.h
+/// \author Marek Bombara
+///
+
+#ifndef QC_MODULE_CTP_CTPRAWDATAQCTASK_H
+#define QC_MODULE_CTP_CTPRAWDATAQCTASK_H
+
+#include "QualityControl/TaskInterface.h"
+
+class TH1F;
+
+using namespace o2::quality_control::core;
+
+namespace o2::quality_control_modules::ctp
+{
+
+/// \brief Example Quality Control DPL Task
+/// \author My Name
+class RawDataQcTask final : public TaskInterface
```

Comment:
Please name the class in a more meaningful way.

Reply:
Would CTPRawDataReaderTask be better?

Reply:
yes